### PR TITLE
[#893] build agains lsp4j snapshots

### PR DESCRIPTION
**Pull request by cdietrich**
[#893] build agains lsp4j snapshots
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: releng/pom.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
 		</repository>
 		<repository>
 			<id>maven</id>
-			<url>${JENKINS_URL}/job/lsp4j/job/master/lastStableBuild/artifact/build/maven-repository</url>
+			<url>http://services.typefox.io/open-source/jenkins/job/lsp4j/job/master/lastStableBuild/artifact/build/maven-repository</url>
```
Comment (cdietrich) [inline releng/pom.xml]: we dont have lsp4j on xtext jipp

Comment (kthoms) [reply releng/pom.xml]: correct

Comment (kthoms) [reply releng/pom.xml]: maybe put this into a separate change, since LSP update not yet mergable

Comment (cdietrich) [reply releng/pom.xml]: no this change builds against snapshots (version change got lost in my previous commit)

Comment (kthoms) [reply releng/pom.xml]: I mean the url change. This should go to master, it is simply wrong.

Comment (cdietrich) [reply releng/pom.xml]: https://github.com/eclipse/xtext-core/pull/954 is meant to be on master now. the release change is at  https://github.com/eclipse/xtext-core/pull/955

Comment (kthoms) [reply releng/pom.xml]: #954 is this one?

Comment (cdietrich) [reply releng/pom.xml]: yes thus 'lsp4j': '0.6.0-SNAPSHOT',

Comment (cdietrich) [reply releng/pom.xml]: => this pr builds against snapshots. the other will set it on release at release say (30.11)

Comment (cdietrich) [reply releng/pom.xml]: @kthoms ok to merge?

**Pull request merged**